client/src/components/Banner/Banner.css

```diff
@@ -18,3 +18,21 @@
   line-height: 72px;
   margin-bottom: 5vh;
 }
+.Banner-booktrip{
+  height: 488px;
+  background-position: center;
+  background-size: cover;
+  width: 100vw;
+  color: #fafafa;
+  font-size: 100px;
+  font-family: var(--default-font);
+  position: relative;
+ 
+}
+.Banner-booktrip h3 {
```

Comment:
What if someone uses an `h2` or `h4`?

Reply:
@ZakkMan  updated this to .Banner--large > *.   This should target the text no matter which tag is used.